src/Bundle/test/app/console

```diff
@@ -19,6 +19,7 @@ use Symfony\Bundle\FrameworkBundle\Console\Application;
 use Symfony\Component\Console\Input\ArgvInput;
 
 $input = new ArgvInput();
+$env = $input->getParameterOption(array('--env', '-e'), getenv('SYMFONY_ENV') ?: 'test');
 
-$application = new Application(new AppKernel('test', true));
+$application = new Application(new AppKernel($env, true));
```

Comment:
What's the idea behind that? Do we really need it?

Reply:
The idea is to test prod requirements (without optional rest bundles).

Reply:
@pamil Maybe you have a better idea to test without optional bundles...